viadot/sources/spectrum.py

```diff
@@ -0,0 +1,166 @@
+from typing import List, Dict, Literal
+import boto3
+
+import pandas as pd
+import awswrangler as wr
+
+
+class Spectrum:
```

Comment:
All sources should inherit from `Source`. Also, should be `RedshiftSpectrum`.

Reply:
Fixed as per [🐛 fixed bug in class inheritance](https://github.com/dyvenia/viadot/pull/579/commits/7aa536096d630b6c4990c21eddc10285efc9ab57)